index.bs

```diff
@@ -1190,6 +1294,39 @@ To <dfn>obtain an event-level report</dfn> given an [=attribution source=] |sour
     :: |trigger|'s [=attribution trigger/debug key=].
 1. Return |report|.
 
+<h3 id="obtaining-required-aggregatable-budget">Obtaining an aggregatable report's required budget</h3>
+
+An [=aggregatable report=] |report|'s <dfn for="aggregatable report">
+required aggregatable budget</dfn> is the total [=aggregatable contribution/value=] of |report|'s
+[=aggregatable report/contributions=].
+
+<h3 algorithm id="obtaining-an-aggregatable-report">Obtaining an aggregatable report</h3>
+
+To <dfn>obtain an aggregatable report</dfn> given an [=attribution source=] |source| and
+an [=attribution trigger=] |trigger|:
+
+1. Let |report| be a new [=aggregatable report=] struct whose items are:
+
+    : [=aggregatable report/reporting endpoint=]
+    :: |source|'s [=attribution source/reporting endpoint=].
+    : [=aggregatable report/attribution destination=]
+    :: |source|'s [=attribution source/attribution destination=].
+    : [=aggregatable report/source time=]
+    :: |source|'s [=attribution source/source time=]
+    : [=aggregatable report/original report time=]
+    :: The result of running [=obtain an aggregatable report delivery time=] with |trigger|'s [=attribution trigger/trigger time=].
+    : [=aggregatable report/report time=]
+    :: |report|'s [=aggregatable report/original report time=]
+    : [=aggregatable report/report id=]
+    :: The result of [=generating a random UUID=].
+    : [=aggregatable report/source debug key=]
+    :: |source|'s [=attribution source/debug key=].
+    : [=aggregatable report/trigger debug key=]
+    :: |trigger|'s [=attribution trigger/debug key=].
+    : [=aggregatable report/contributions=]
+    :: The result of running [=create aggregatable contributions=] with |source| and |trigger|.
```

Comment:
Either all of these `:: <value>` should end in `.` or none of them should.

Reply:
Add `.` to all values. Thanks.